sys/vm/vm_map.c

```diff
@@ -5058,9 +5062,9 @@ vmspace_fork(struct vmspace *vm1, vm_ooffset_t *fork_charge)
 	 * XXX NWF We should probably go around again to force the epoch closed.
 	 */
 	vm2->vm_map.vm_cheri_revoke_st = vm1->vm_map.vm_cheri_revoke_st;
+	vm2->vm_map.vm_cheri_revoke_test = vm1->vm_map.vm_cheri_revoke_test;
```

Comment:
At the moment, at least, we should never get here with a test installed (part of the "no state transition in progress" bit of the comment above).  Assert that, instead?

Reply:
Ah, no, sorry, you're right.  The whole "no state transition in progress" thing is a holdover from Cornucopia pre-Reloaded.  Under Reloaded we can't get here with a thread doing sweeping, but we can get here with an epoch open and load faults happening.  Thanks for the (unintentional) reminder on Slack.

Reply:
It's worth noting that we don't get here in current testing (I'd added an assert before realizing it was wrong). We need to fix that.